<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Login</class>
 <widget class="QDialog" name="Dialog_Login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login do Sistema</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Usu√°rio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtNome"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Senha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSenha">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Lembra da Senha</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEntrar">
       <property name="text">
        <string>Entrar</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/imagens/icons/1493238513_Check.png</normaloff>:/imagens/icons/1493238513_Check.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Sair</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/imagens/icons/1493332785_button_cancel.png</normaloff>:/imagens/icons/1493332785_button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
